examples/textbooks/readme.md

```diff
@@ -1,5 +1,72 @@
-# Installation
+# Guidelines
 
-Install inference-swarm and the requirements in the example.
 
-Install `datatrove` from source
```

Comment:
Actually, maybe you could keep the `datatrove` installation instruction still here? I don't think the PyPi release of `datatrove` is the latest

Reply:
added it back (it's not needed for my example but it is for  `generate_rw_textbooks.py`, will update it later)